scipy/signal/_fir_filter_design.py

```diff
@@ -1280,3 +1281,123 @@ def minimum_phase(h: np.ndarray,
         h_minimum = h_temp.real
     n_out = (n_half + len(h) % 2) if half else len(h)
     return h_minimum[:n_out]
+
+
+def fwind1(hsize, window, *, fc=None, fs=2, circular=False):
+    """
+    2D FIR filter design using the window method.
+
+    This function computes the coefficients of a 2D finite impulse response
+    filter. The filter is separable with linear phase; it will be designed
+    as a product of two 1D filters with dimensions defined by `hsize`.
+    Additionally, it can create approximately circularly symmetric 2-D windows.
+
+    Parameters
+    ----------
+    hsize : tuple or list of length 2
+        Lengths of the filter in each dimension. `hsize[0]` specifies the
+        number of coefficients in the row direction and `hsize[1]` specifies
+        the number of coefficients in the column direction.
+    window : tuple or list of length 2 or string
+        Desired window to use for each 1D filter or a single window type 
+        for creating circularly symmetric 2-D windows. Each element should be
+        a string or tuple of string and parameter values. See
+        '~scipy.signal.get_window' for a list of windows and required
+        parameters.
+    fc : float or 1-D array_like, optional
+        Cutoff frequency of filter (expressed in the same units as `fs`).
+        Required if `circular` is False.
+    fs : float, optional
+        The sampling frequency of the signal. Default is 2.
+    circular : bool, optional
+        Whether to create a circularly symmetric 2-D window. Default is False.
+
+    Returns
+    -------
+    filter_2d : (hsize[0], hsize[1]) ndarray
+        Coefficients of 2D FIR filter.
+
+    Raises
+    ------
+    ValueError
+        If `hsize` and `window` are not 2-element tuples or lists.
+        If `cutoff` is None when `circular` is True.
+        If `cutoff` is outside the range [0, fs / 2] and `circular` is False.
+        If any of the elements in `window` are not recognized.
+    RuntimeError
+        If `firwin` fails to converge when designing the filter.
+
+    See Also
+    --------
+    scipy.signal.firwin, scipy.signal.get_window
+
+    Examples
+    --------
+    Generate a 5x5 low-pass filter with cutoff frequency 0.1.
+
+    >>> import numpy as np
+    >>> from scipy.signal import get_window
+    >>> from scipy.signal import fwind1
+    >>> hsize = (5, 5)
+    >>> window = (("kaiser", 5.0), ("kaiser", 5.0))
+    >>> fc = 0.1
+    >>> filter_2d = fwind1(hsize, window)
```

Comment:
Rather than removing fc
```suggestion
    >>> filter_2d = fwind1(hsize, window, fc=fc)
```